### DISCUSSION TITLE
Mail send

### DISCUSSION THREAD

**Discussion by nokiokid** [PHPMailer/PHPMailer#2818] [General]
Please excuse me if I'm in the wrong section, I;m new to all this, just need a bit of help. I run a website and have just set up a competition entry form, all goes Ok but the email function doesn't appear to work and I get this error 
https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting
Now this has brought me to this forum. I have followed the link but don't have a clue how to carry on. On my web site I have set up sendmail to send emails and using the send test email button works OK, I get this message from my email account 
This is a test mail sent using "Sendmail". Your email settings are correct!

**Comment by Synchro** [comment]
That URL isn't an error - but it will be displayed after the error, so you need to tell us the rest of what it says. Please post your code, and do what the troubleshooting guide says about showing debug output, as that will show what's going on.

**Reply by nokiokid** [reply]
This is the full warning  Mailer Error: SMTP connect() failed. https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting

**Reply by Synchro** [reply]
There's nothing I can say here that's not already covered [in the troubleshooting guide on this exact subject](https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting#smtp-error-could-not-connect-to-smtp-host).
